config.toml

```diff
@@ -28,6 +28,21 @@ defaultContentLanguage = "en"
     "app"
   ]
 
+[[params.versions]]
+version = "v3.0.0"
+patch = "beta.2"
+url = "https://v3.helm.sh/docs"
+primary = true
+
+[[params.versions]]
+version = "v2.14.3"
```

Comment:
Would it be best to call this version "latest" instead? That way we don't have to make a new PR to bump it every time a new release is out.

Reply:
Good idea - that would simplify the site handling of versions. 

_Current live site_:

<img width="252" alt="Screen Shot 2019-09-04 at 9 19 13 AM" src="https://user-images.githubusercontent.com/686194/64272895-2257ce80-cef5-11e9-852f-a71cbe0c1850.png">

_Simpler labels suggestion_:

<img width="231" alt="Screen Shot 2019-09-04 at 9 27 10 AM" src="https://user-images.githubusercontent.com/686194/64273591-77e0ab00-cef6-11e9-9dd3-ceeb3e01cda2.png">
 
---

I worry that latest is too vague when we are presenting v2 and v3 choices. The version number is significant as the docs are very different between the two, and I think clarifying whether it's alpha, beta or RC helps show how close v3 is to becoming 'latest'. 

I'd like to keep manually adding as much clarification info as possible for now, and switch to having the top, default choice read `latest` when 3 is released. It'll be much simpler then.